common/common.cpp

```diff
@@ -768,7 +768,11 @@ bool gpt_params_find_arg(int argc, char ** argv, const std::string & arg, gpt_pa
             invalid_param = true;
             return true;
         }
-        params.image = argv[i];
+        if (params.image != "") {
+            params.image += ",";
+        }
+        params.image += argv[i];
+        params.image = std::regex_replace(params.image,std::regex(" --image "), ",");
```

Comment:
It seems that gpt_params::image has to ge std::vector<std::string> to hold multiple images.
also, argv[i] will not contain the string " --image " as `++i` was stated beforehand

Reply:
I've rewritten with a vector of std::string datatypes, looking at other existing examples